setup.sh

```diff
@@ -1,44 +1,39 @@
 #!/usr/bin/env bash
 set -e
 
-source ${BASH_SOURCE%/*}/vault_auth.sh
+HERE=${BASH_SOURCE%/*}
+source ${HERE}/vault_auth.sh
 
 echo "Installing Pip and required Python packages"
 apt-get install python3-pip -y > /dev/null
-pip3 install -q -r requirements.txt
+pip3 install -q -r ${HERE}/requirements.txt
 
 echo "-------------------------------------------"
 echo "Installing bb8 user and group"
 
 if ! id -u bb8 > /dev/null 2>&1; then
-    useradd bb8
+    useradd bb8 -d /usr/lib/bb8
     password=$(vault read --field password secret/backup/bb8/user)
     echo "bb8:$password" | chpasswd
 
-    # set bb8 home dir
-    mkdir /var/lib/bb8/.ssh
-    usermod -m -d /var/lib/bb8 bb8
-    chown -R bb8:bb8 /var/lib/bb8
+    # Perhaps defer until python?
+    mkdir -p /var/lib/bb8/.ssh
 
     # add to docker group
     usermod -aG docker bb8
 fi
 
-if ! /usr/bin/getent group bb8 2>&1 > /dev/null; then
-    groupadd bb8
-fi
```

Comment:
isn't this needed?

Reply:
Not from what I could see running it in the container.  The group is created by useradd (see `man useradd`)

>        By default, a group will also be created for the new user (see -g, -N,
>       -U, and USERGROUPS_ENAB).

Reply:
you're totally right; cool